library/src/main/java/com/cube/storm/ui/quiz/view/holder/ImageQuizItemViewHolder.java

```diff
@@ -80,10 +80,16 @@ public ImageQuizItemViewHolder(View view)
 
 			if (index < images.size())
 			{
-				((ImageView)currentCell.findViewById(R.id.image)).populate(model.getImages().get(index));
-				((Checkable)currentCell.findViewById(R.id.checkbox)).setChecked(model.getSelectHistory().contains(index));
-				currentCell.setTag(R.id.checkbox, index);
-				currentCell.setOnClickListener(new ModelClickListener(model));
+				answerLayouts.add(index, currentCell);
+				ImageView imageView = currentCell.findViewById(R.id.image);
+				imageView.populate(model.getImages().get(index));
+
+				if (model.getSelectHistory().contains(index))
+				{
+					selectAnswer(currentCell, true);
```

Comment:
I think you'll need to use `selectAnswer(currentCell, false)` in the `else` branch just in case this view is recycled?